src/pages/Graph/SummaryPanelEdge.tsx

```diff
@@ -411,6 +417,15 @@ export default class SummaryPanelEdge extends React.Component<SummaryPanelPropTy
       );
     }
 
+    const data = nodeData(edge.target());
+    if (data.isInaccessible) {
+      return (
+        <>
+          <Icon type="pf" name="info" /> Sparkline charts cannot be shown because the destination is inaccessible.
+        </>
+      );
+    }
+
```

Comment:
Again, I'm not sure this is true.  Maybe I am wrong, but I would expect to be able to see the edge traffic to an inaccessible node.

Reply:
Well... Tried this with a more "real" scenario. What I did is to deploy a traffic generator to a side "myproject" namespace and make a request to productpage in bookinfo namespace. Then, configured Kiali with reduced permissions giving access only to "myproject". So, I get this graph "myproject" namespace:

![image](https://user-images.githubusercontent.com/23639005/55096268-2dbde880-507f-11e9-8a63-166b4bc20f77.png)

So, _productpage_ is inaccessible. What I see is that when I select the traffic-generator -> productpage edge, the request that goes to the backend is:

`http://localhost:3000/api/namespaces/bookinfo/services/productpage/metrics?queryTime=1553705660&duration=60&step=30&rateInterval=30s&filters[]=request_count&filters[]=request_duration&filters[]=request_error_count&quantiles[]=0.5&quantiles[]=0.95&quantiles[]=0.99&byLabels[]=source_workload&direction=inbound&reporter=source&requestProtocol=http`

You can note that the request asks for data in the `bookinfo` namespace. But that one is inaccessible. So, the back-end correctly replies with an error:

```js
{
  "error":
     "Cannot access namespace data: projects.project.openshift.io \"bookinfo\" is forbidden: User \"system:serviceaccount:istio-system:kiali-service-account\" cannot get projects.project.openshift.io in the namespace \"bookinfo\": no RBAC policy matched"}
```

So, I don't see this wrong since we cannot query the data for the sparklines. Though, the other data of the side panel is available:

![image](https://user-images.githubusercontent.com/23639005/55097645-dec58280-5081-11e9-8616-70a68a85848e.png)

Reply:
Well... Tried this with a more "real" scenario. What I did is to deploy a traffic generator to a side "myproject" namespace and make a request to productpage in bookinfo namespace. Then, configured Kiali with reduced permissions giving access only to "myproject". So, I get this graph "myproject" namespace:

![image](https://user-images.githubusercontent.com/23639005/55096268-2dbde880-507f-11e9-8a63-166b4bc20f77.png)

So, _productpage_ is inaccessible. What I see is that when I select the traffic-generator -> productpage edge, the request that goes to the backend is:

`http://localhost:3000/api/namespaces/bookinfo/services/productpage/metrics?queryTime=1553705660&duration=60&step=30&rateInterval=30s&filters[]=request_count&filters[]=request_duration&filters[]=request_error_count&quantiles[]=0.5&quantiles[]=0.95&quantiles[]=0.99&byLabels[]=source_workload&direction=inbound&reporter=source&requestProtocol=http`

You can note that the request asks for data in the `bookinfo` namespace. But that one is inaccessible. So, the back-end correctly replies with an error:

```js
{
  "error":
     "Cannot access namespace data: projects.project.openshift.io \"bookinfo\" is forbidden: User \"system:serviceaccount:istio-system:kiali-service-account\" cannot get projects.project.openshift.io in the namespace \"bookinfo\": no RBAC policy matched"}
```

So, I don't see this wrong since we cannot query the data for the sparklines. Though, the other data of the side panel is available:

![image](https://user-images.githubusercontent.com/23639005/55097645-dec58280-5081-11e9-8616-70a68a85848e.png)